src/Keyboard.css

```diff
@@ -51,7 +51,8 @@
 	flex-basis: 56px;
 }
 
-.keyboard-space {
+.keyboard-space,
+.keyboard-spaceButton{
```

Comment:
I agree BEM would be ideal, but that would mean breaking changes for anyone who used the project and hooked into the CSS classes.